Dockerfile

```diff
@@ -21,26 +19,22 @@ COPY --chown=sydent:sydent ["sydent", "/sydent/sydent"]
 COPY --chown=sydent:sydent ["README.rst", "setup.cfg", "setup.py", "/sydent/"]
 
 # Install dependencies
-RUN cd /sydent \
-    && su sydent -c 'pip install --user --upgrade pip setuptools sentry-sdk prometheus_client' \
-    && su sydent -c 'pip install --user -e .' \
-    && rm -rf /sydent/.cache \
-    && find /sydent -name '*.pyc' -delete
+USER sydent
+WORKDIR /sydent
+RUN pip install --user --upgrade pip setuptools sentry-sdk prometheus_client
+RUN pip install --user .
+RUN rm -rf /sydent/.cache
+RUN find /sydent -name '*.pyc' -delete
```

Comment:
We should probably keep these in a single `RUN` block, since each step creates a new filesystem layer. Even though we delete the caches and `pyc` files in the final two steps, they're still present in the prior layers and have to be stored / downloaded. This negates any benefit from the final `rm` and `-delete` steps.

Reply:
Ah! I didn't realize that's why it was being done this way. Seems like that should be an explicit step, but...Docker.

Reply:
(Also, it makes it a huge pain to debug...)

Reply:
@callahad I made this change although it isn't clear to me if the `USER` and `WORKDIR` steps also should still be in there.

Reply:
I think at that point it's more up to personal preference :)